### ISSUE TITLE
[BUG] 'Disable for Non-Hostile' disables Conditional Visability's hiding of tokens for all tokens

### ISSUE THREAD

**Issue by Brimcon** [p4535992/conditional-visibility#23]
**Module Version:** v0.5.15
**Foundry Version:** 9.255
**DnD5e Version:** 1.5.7

**Describe the bug**
With `Disable for non hostile npc` enabled. Selecting a Hostile NPC (In this case, Acolyte from DnD5e) still shows the hidden/invisible token.

**To Reproduce**
With only Conditional Visibility and its dependencies enabled import from DnD5e three actors. Two "Starter Heroes" (Suggest Using the Dwarf Rogue for 'higher' rolls on stealth, the second hero does not matter) as well as a Monster (Acolyte is the easiest as it has a very low Passive Perception)
Enable `Disable for non hostile npc` in Module Settings
Enable `Auto hide on stealth rolls` to make the stealth check more convenient
Roll a Stealth Check for one of the Heroes. (Again, Dwarf Rogue has a higher score)
Click on the Hostile Token.
Observe "Hidden" Actor still visible.

**Expected behavior**
Hostile tokens unable to see the player token.

**Screenshots**
![image](https://user-images.githubusercontent.com/7198283/160248372-2453d50e-665c-47a2-a884-d3b50a2ca784.png)

![image](https://user-images.githubusercontent.com/7198283/160248397-c40b06f1-0276-4d4a-81fb-ad4afb3289f3.png)

**Browser:**
Chrome

**Additional context**
[CVHostile Enabled.log](https://github.com/p4535992/conditional-visibility/files/8355867/CVHostile.Enabled.log)

[CVHostile Disabled.log](https://github.com/p4535992/conditional-visibility/files/8355869/CVHostile.Disabled.log)

Edit: This error pops up on a Player Client when the Hostile token rolls for Stealth
```js
libwrapper.js:646 Uncaught (in promise) Error: User Player4 lacks permission to update Token [sXXzZmRxigdBzDiW] in parent Scene [Nu75iPQRJrJ2YCoo]
[No packages detected]
    at ServerDatabaseBackend._updateEmbeddedDocuments (/home/foundry/resources/app/dist/database/backend/server-backend.mjs:1:3611)
    at async Semaphore._try (/home/foundry/resources/app/common/utils/semaphore.mjs:99:17)
```

**Comment by p4535992** [comment]
Should be solved with 0.5.20, let me know

**Comment by Brimcon** [comment]
It is fixed! But this error still prints if an NPC or even another player rolls stealth with the auto stealth function enabled:

```js
Uncaught (in promise) Error: User Player3 lacks permission to update Token [sXXzZmRxigdBzDiW] in parent Scene [Nu75iPQRJrJ2YCoo]
[No packages detected]
    at ServerDatabaseBackend._updateEmbeddedDocuments (/home/foundry/resources/app/dist/database/backend/server-backend.mjs:1:3611)
    at async Semaphore._try (/home/foundry/resources/app/common/utils/semaphore.mjs:99:17)
```

But that might need a different Issue Report and outside the scope of this one.
